src/openfermion/hamiltonians/_molecular_data.py

```diff
@@ -820,94 +820,64 @@ def get_active_space_integrals(self,
                                              active_indices,
                                              active_indices)])
 
-    def get_molecular_hamiltonian(self,
-                                  occupied_indices=None,
-                                  active_indices=None,
-                                  spin_indexing="even-odd"):
+    def get_molecular_hamiltonian(self,occupied_indices=None,
+                                      active_indices=None):
```

Comment:
this spacing seems to be inconsistent with pep8. No?

Reply:
indeed. I'm failing at pep8ing with this commit.

Reply:
I figured we had given you enough trouble. Perhaps clean it up on the next PR if you're feeling charitable ;-)